cleanlab/datalab/issue_manager/label.py

```diff
@@ -118,9 +126,38 @@ def _reset(self) -> None:
 
     def find_issues(
         self,
-        pred_probs: np.ndarray,
+        pred_probs: Optional[npt.NDArray] = None,
+        features: Optional[npt.NDArray] = None,
         **kwargs,
     ) -> None:
+        """Find label issues in the datalab.
+
+        Parameters
+        ----------
+        pred_probs :
+            The predicted probabilities for each example.
+
+        features :
+            The features for each example.
+        """
+        if pred_probs is None:
+            if features is None:
+                raise ValueError(
+                    "Either pred_probs or features must be provided to find label issues."
+                )
+            # produce out-of-sample pred_probs from features
+            knn = KNeighborsClassifier(n_neighbors=self.k)
+            knn.fit(features, self.datalab.labels)
+            pred_probs = knn.predict_proba(features)
+
+            encoder = OneHotEncoder()
+            label_transform = self.datalab.labels.reshape(-1, 1)
+            one_hot_label = encoder.fit_transform(label_transform)
+
+            pred_probs = np.asarray(
+                (pred_probs - 1 / self.k * one_hot_label) * self.k / (self.k - 1)
+            )
```

Comment:
I think I am doing the out-of-sample `pred_probs` here according to the method described in the earlier comment https://github.com/cleanlab/cleanlab/pull/760#discussion_r1256490110

Reply:
Ah yes my bad sorry, thanks for pointing out! Can just keep the comment as is.